app/models/null_service_provider.rb

```diff
@@ -31,5 +31,5 @@ def friendly_name; end
 
   def return_to_sp_url; end
 
-  def redirect_uri; end
+  def redirect_uris; end
```

Comment:
Looks like the failing spec is expecting this to return an empty array, not `nil`.